src/e2e-test/java/io/cdap/plugin/bqexecute/stepsdesign/BQExecuteConnector.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.bqexecute.stepsdesign;
+
+import io.cdap.e2e.pages.actions.CdfStudioActions;
+import io.cdap.e2e.pages.locators.CdfStudioLocators;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.bqexecute.actions.CdfBQExecuteActions;
+import io.cdap.plugin.common.stepsdesign.TestSetupHooks;
+import io.cdap.plugin.utils.E2EHelper;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import java.io.IOException;
+
+/**
+ * BigQuery Execute Connector related Step Design.
+ */
+public class BQExecuteConnector implements E2EHelper {
+
+    @When("Select BigQuery Execute Connector")
+    public void selectBigQueryExecuteConnector() throws InterruptedException {
+        CdfBQExecuteActions.selectCondiationandAction();
+        CdfBQExecuteActions.selectBQExecute();
+    }
+
+    @Then("Open BigQuery Execute connector properties")
```

Comment:
Remove this.
Use  -> When Navigate to the properties page of plugin

Reply:
@sawantpritam -done